azext_iot/central/_help.py

```diff
@@ -44,6 +57,288 @@ def load_central_help():
+                'container': 'test'
+              }
+            }"
+
+        - name: create a Azure Data Explorer export destination with json payload
+          text: >
+            az iot central export destination create
+            --app-id {appid}
+            --dest-id {destintionid}
+            --content "{
+              'displayName': 'Azure Data Explorer',
+              'type': 'dataexplorer@v1',
+              'clusterUrl': 'https://[clusterName].westus2.kusto.windows.net',
+              'database': 'database1',
+              'table': 'table1',
+              'authorization': {
+                'type': 'servicePrincipal',
+                'clientId': '3b420743-2020-44c6-9b70-cc42f945db0x',
+                'tenantId': '72f988bf-86f1-41af-91ab-2d7cd011db47',
+                'clientSecret': '[Secret]'
+              }
+            }"
+
+        - name: create an Event Hub export destination with json payload
+          text: >
+            az iot central export destination create
+            --app-id {appid}
+            --dest-id {destintionid}
+            --content "{
+              'displayName': 'Event Hub',
+              'type': 'eventhubs@v1',
+              'authorization': {
+                'type': 'connectionString',
+                'connectionString': 'Endpoint=sb://[hubName].servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=*****;EntityPath=entityPath1'
+              }
+            }"
+
+        - name: create an Service Bus Queue destination with json payload
+          text: >
+            az iot central export destination create
+            --app-id {appid}
+            --dest-id {destintionid}
+            --content "{
+              'displayName': 'Service Bus Queue',
+              'type': 'servicebusqueue@v1',
+              'authorization': {
+                'type': 'connectionString',
+                'connectionString': 'Endpoint=sb://[namespance].servicebus.windows.net/;SharedAccessKeyName=xxx;SharedAccessKey=[key];EntityPath=[name]'
+              }
+            }"
+
+        - name: create an Service Bus Topic destination with json payload
+          text: >
+            az iot central export destination create
+            --app-id {appid}
+            --dest-id {destintionid}
+            --content "{
+              'displayName': 'Service Bus Topic',
+              'type': 'servicebustopic@v1',
+              'authorization': {
+                'type': 'connectionString',
+                'connectionString': 'Endpoint=sb://[namespace].servicebus.windows.net/;SharedAccessKeyName=xxx;SharedAccessKey=[key];EntityPath=[name]'
+              }
+            }"
+    """
+
+    helps[
+        "iot central export destination update"
+    ] = """
+        type: command
+        short-summary: Update an export destination for an IoT Central application.
```

Comment:
Can you change the type of the export here? If not, this may be worth mentioning in help

Reply:
Add extra comments on "Source" are not allowed to be updated for export, "Type" are not allow to be updated for destination.